.azure-pipelines/common-templates/download-openapi-docs.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           script: |
             git status
             git add .
-            git commit -m 'Weekly OpenApiDocs Download'
+            git commit -m 'Weekly OpenApiDocs Download. [run ci]'
```

Comment:
What's the impact of this?

Reply:
It should trigger a GitHub action to create a PR when changes are detected - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/.github/workflows/create-openapidocs-pr-workflow.yml#L16.